Tests/NIOFileSystemIntegrationTests/FileHandleTests.swift

```diff
@@ -19,7 +19,7 @@ import NIOPosix
 import NIOFoundationCompat
 import XCTest
 
-@available(macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0, *)
+@available(macOS 10.15, iOS 15.0, watchOS 6.0, tvOS 13.0, *)
```

Comment:
Is this the right bump, is it just iOS affected?

Reply:
You're right, I was missing bumping the other platforms.